x-pack/plugins/data_visualizer/public/application/common/components/stats_table/_index.scss

```diff
@@ -46,11 +66,41 @@
       display: none;
     }
   }
+
   .dataVisualizerSummaryTableWrapper {
-    max-width: 300px;
+    min-width: $panelWidthS;
+    max-width: $panelWidthS;
+    max-height: 120px;
   }
+
   .dataVisualizerMapWrapper {
     min-height: 300px;
-    min-width: 600px;
+    min-width: $panelWidthS;
+    max-width: $panelWidthS;
+  }
+
+  .dataVisualizerTopValuesWrapper {
+    min-width: $panelWidthS;
+    max-width: $panelWidthL;
+  }
+
+  .dataVisualizerUniformPanel {
+    min-width: $panelWidthS;
+    max-width: $panelWidthS;
   }
+
+  .dataVisualizerPanelWrapper {
+    border-radius: $euiBorderRadius;
+    border: 1px solid $euiBorderColor;
+    padding: 6px;
```

Comment:
You can use the existing **EuiPanel** component and then don't have to set these properties.